pkg/kubelet/dockertools/manager_test.go

```diff
@@ -214,3 +189,22 @@ func TestGetPods(t *testing.T) {
 		t.Errorf("expected %#v, got %#v", expected, actual)
 	}
 }
+
+func TestListImages(t *testing.T) {
+	manager, fakeDocker := NewFakeDockerManager()
+	dockerImages := []docker.APIImages{{ID: "1111"}, {ID: "2222"}, {ID: "3333"}}
+	expected := util.NewStringSet([]string{"1111", "2222", "3333"}...)
+
+	fakeDocker.Images = dockerImages
+	actualImages, err := manager.ListImages()
+	if err != nil {
+		t.Fatalf("unexpected error %v", err)
+	}
+	actual := util.NewStringSet()
+	for _, i := range actualImages {
+		actual.Insert(i.ID)
+	}
+	if !reflect.DeepEqual(expected.List(), actual.List()) {
```

Comment:
It is. util.StringSet sorts before returns.
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/util/set.go#L122